HW17/src/calculator.ts

```diff
@@ -0,0 +1,24 @@
+export class Calculator {
+    constructor() {};
+    public numbersAdder(firstNumber: number, secondNumber: number, operation: '+') {
```

Comment:
Please rename: `adder`.
It is obvious that `Calcilator` deals with numbers. The same is for other methods.